core/src/main/java/org/folio/tlib/util/TenantUtil.java

```diff
@@ -24,6 +24,10 @@ public static String tenant(RoutingContext ctx) {
     if (tenant == null) {
       throw new IllegalArgumentException(XOkapiHeaders.TENANT + " header is missing");
     }
+    if (tenant.length() > 31) {
+      throw new IllegalArgumentException(
+          XOkapiHeaders.TENANT + " header size must not be greater than 31");
+    }
```

Comment:
This length restriction should be removed because Okapi currently doesn't have any length restriction.

We can add a length restriction at a later time.

Reply:
ok